Documentation/Getting-Started/storage-architecture.md

```diff
@@ -21,12 +21,89 @@ specified in the Ceph custom resources (CRs) and apply the changes.
 
 Rook automatically configures the Ceph-CSI driver to mount the storage to your pods.
 
-![Rook Components on Kubernetes](ceph-storage/kubernetes.png)
+![Rook Components on Kubernetes](ceph-storage/Rook High-Level Architecture.png)
 
+Example applications are shown above for the three supported storage types:
+
+- Block Storage is represented with a blue app, which has a `ReadWriteOnce (RWO)` volume mounted. The application can read and write to the RWO volume, while Ceph manages the IO.
+- Shared Filesystem is represented by two purple apps that are sharing a ReadWriteMany (RWX) volume. Both applications can actively read or write simultaneously to the volume. Ceph will ensure the data is safely protected for multiple writers with the MDS daemon.
+- Object storage is represented by an orange app that can read and write to a bucket with a standard S3 client.
+
+Below the dotted line in the above diagram, the components fall into three categories:
+
+- Rook operator (blue layer): The operator automates configuration of Ceph
+- CSI plugins and provisioners (orange layer): The Ceph-CSI driver provides the provisioning and mounting of volumes
+- Ceph daemons (red layer): The Ceph daemons run the core storage architecture. See the [Glossary](glossary.md#ceph) to learn more about each daemon.
+
+Production clusters must have three or more nodes for a resilient storage platform.
+
+## Diagrams
```

Comment:
Shouldn't this Diagrams title be down on line 52? These paragraphs before that aren't related to the diagrams

Reply:
@travisn Like this? Might have miscopied while I tried to fix the paragraphs move

Reply:
Looks good now, thanks